enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/components/SdkIframeEmbedRoute.tsx

```diff
@@ -50,11 +64,20 @@ export const SdkIframeEmbedRoute = () => {
   });
 
   return (
-    <MetabaseProvider authConfig={authConfig} theme={theme} locale={locale}>
-      <Box h="100vh" bg={theme?.colors?.background}>
-        <SdkIframeEmbedView settings={embedSettings} />
-      </Box>
-    </MetabaseProvider>
+    <MetabaseReduxProvider store={storeRef.current}>
```

Comment:
We need access to the store to `setIsSdkIframeEmbedAuth(true)`

Reply:
We won't if we just use an environment variable 😉